.travis.yml

```diff
@@ -95,6 +96,7 @@ script:
   - python labcontrol/gui/js_tests/js_tester.py all-tests
   - nosetests --exclude-dir=labcontrol/gui/js_tests --with-doctest --with-coverage -v --cover-package=labcontrol
   - flake8 labman setup.py scripts/*
+  - make festylecheck
```

Comment:
if anything, add another comment here saying this like performs the style check; since it's not the same name as prettier, it's not immediately obvious to anyone who isn't seeing the two lines highlighted.